cdci_data_analysis/analysis/parameters.py

```diff
@@ -302,10 +302,18 @@ def set_value_from_form(self, form, verbose=False):
     def set_par(self, value, units=None):
         if units is not None:
             self.units = units
+
         self.value = value
-        return self.get_value_in_default_format()
+        return self.get_value_in_default_units()
+
+    def get_value_in_default_units(self):
+        return self.value
```

Comment:
does it make sense if  this calls `get_value_in_units`?

Reply:
well in this case yes

Reply:
and therefore avoid repeating in inherited classes